StreamChatUITestsAppUITests/Tests/Performance/LoadTime.swift

```diff
@@ -0,0 +1,42 @@
+//
+// Copyright © 2024 Stream.io Inc. All rights reserved.
+//
+
+import XCTest
+
+@available(iOS 15.0, *)
+class LoadTime: StreamTestCase {
+    
+    func testMessageList() {
+        WHEN("user logs in") {
+            backendRobot.generateChannels(count: 1, messagesCount: 200)
```

Comment:
What do the robots generate? Is it objects directly in memory or is decoding included? Since decoding takes the most of the time when loading new data.

Reply:
The BackendRobot generates a json that will be returned to the SDK on `/channels/*` call

Reply:
great, that's what we need